app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
```

Comment:
Kindly, make sure `sign up` is working as expected when I try to create a new user I'm getting this error 👇 

<img width="1645" alt="image" src="https://user-images.githubusercontent.com/34578631/174039264-f4531d57-2a02-4abc-83fa-50b3e7dc9e76.png">